scripts/server

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${ASHLAR_BLUEPRINT_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n "Usage: $(basename "$0")
+
+Starts servers using docker-compose.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        if [[ $(sudo netstat -nlp | grep :::8000) ]];
+        then
+            echo "It appears you already have a process running on port 8000:"
+            echo
+            echo $(sudo netstat -nlp | grep :::8000)
```

Comment:
I love this! I wish everything would do this when there are port conflicts because I always have to google for the correct netstat command if I don't immediately know what's causing the conflict.